README.md

```diff
@@ -126,40 +153,55 @@ python src/lbaf/Applications/LBAF_app.py -c <config-file-name> -v <verbosity-lev
 
 To output only the Rank QOI, use `-v 1`. Otherwise, to print both Rank and Object QOI, use `-v 2`.
 
-## JSON data files Validator
+## Additional Programs
+
+<details>
+<summary><b> JSON data files validator</b></summary>
+
+<br />
 
 JSON data files Validator validates VT data files against defined schema. It is located in the VT repository and can be found [here](https://raw.githubusercontent.com/DARMA-tasking/vt/develop/scripts/JSON_data_files_validator.py).
 
+#### Download into LBAF
 
-### Download into LBAF
+If the `lbaf` package is installed, run:
 
-To run using the lbaf package:
 ```shell
 lbaf-vt-data-files-validator-loader
 ```
-To run from source:
+
+Otherwise, run from source:
+
 ```shell
 cd <project-path>
 python src/lbaf/Utils/lbsJSONDataFilesValidatorLoader.py
 ```
 
-The script is then saved to `<project-path>/src/lbaf/imported/JSON_data_files_validator.py`
+The script will be saved to `<project-path>/src/lbaf/imported/JSON_data_files_validator.py`
 
-### Download and Run from LBAF
+#### Run from LBAF
+
+If the `lbaf` package is installed, run:
 
-To run using the lbaf package:
 ```shell
 lbaf-vt-data-files-validator
 ```
-To run from source:
+
+Otherwise, run from source:
+
 ```shell
 cd <project-path>
 python src/lbaf/imported/JSON_data_files_validator.py
 ```
-This command automatically downloads the JSON_data_files_validator.py script if needed.
 
+_Note: This command automatically downloads the `JSON_data_files_validator.py` script if needed._
+
+#### Usage
+
+These commands assume that LBAF was installed as a package. If you are running from source, you will need to replace the run command as noted above.
```

Comment:
`When running from source, replace the run command...`

Reply:
Fixed